packages/config/config/devices/0x0258/neo-ds07z.json

```
@@ -0,0 +1,158 @@
+		"reset": "1. Power Up The Device.\n2. Press And Hold The Button For 10s Until Pink Led Lights Is On, Then Release The Button Before Led Turn Off."
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Led Indicated",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Binary Sensor Report",
+			"description": "Enable SENSOR_BINARY_REPORT When Device Detects A Door/Window Opened Or Closed Event.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Temperature Offset Value",
+			"description": "The Current Measuring Temperature Value Can Be Add And Minus A Value By This Setting. Temperature Offset Value = [Value] × 0.1 Degree Celsius / Fahrenheit (Us).Temperature Offset Value = [Value] × 0.1 Degree Celsius / Fahrenheit (Us).",
+			"valueSize": 1,
+			"minValue": -120,
+			"maxValue": 121,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "4",
+			"label": "Humidity Offset Value",
+			"description": "The current measuring humidity value can be add and minus a value by this setting. Humidity Offset Value = [Value] × 0.1 RH%.",
+			"valueSize": 1,
+			"minValue": -120,
+			"maxValue": 120,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "5",
+			"label": "Temperature D-Value Setting",
+			"description": "Sets The Changed Value Of The Temperature. When The Difference From The Last Report Exceeds This Setting Value, The Device Will Report Current Temperature Value To Z-Wave Hubs. The D-Value = [Value] × 0.1 Degree Celsius / Fahrenheit (Us).",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "Humidity D-Value Setting",
+			"description": "Sets The Changed Value Of The Humidity. When The Difference From The Last Report Exceeds This Setting Value, The Device Will Report Current Humidity Value To Z-Wave Hubs. The D- Value = [Value] × 0.1 Rh%.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 20,
+			"unsigned": true
+		},
+		{
+			"#": "7",
+			"label": "Basic Set Value",
+			"description": "Sets The Level For Device Sending BASIC_SET To Nodes That Associated In Group 2 When Device Detects A Door/Window Opened Event.  [1 … 99] On. - Basic_Set = [Setting Value].",
```

For consistency with other files, this should be
```suggestion
			"label": "Association Group 2: Basic Set Value (Open)",
```